pkgs/tools/misc/colord-kde/default.nix

```diff
@@ -1,11 +1,11 @@
-{ stdenv, lib, fetchurl
+{ mkDerivation, stdenv, lib, fetchurl
```

Comment:
There's no other uses of `stdenv` in this function, so I guess it could be dropped from the arguments?

Or should it use `stdenv.lib` instead of the `lib` argument?